wgsl/index.bs

```diff
@@ -3844,7 +3844,7 @@ Issue: (mehmetoguzderin) Refine precedence rules ....
       <b>3</b>
     <td> <b><code>*</code></b>
     <td> Multiplication
-    <td colspan='1' rowspan='0' style='text-align: left'>
+    <td colspan='1' rowspan='18' style='text-align: left'>
```

Comment:
Why does this span 18 rows?

Reply:
@dj2 Because those are all left-to-right and some browsers only render when is 18.

Reply:
If you look at https://mehmetoguzderin.github.io/webgpu/webgpu-20200925-precedence.html#precedence it should be clear.